repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/model/AccessTypeRolesJurisdictionResult.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.ccd.definition.store.repository.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@ApiModel(description = "jurisdiction")
+public class AccessTypeRolesJurisdictionResult {
+
+    private String id = null;
```

Comment:
might be better to name these fields `jurisdictionId` & `jurisdictionName`, that way no need to use `@JsonProperty("jurisdictionId")` below